post-start.sh

```diff
@@ -18,18 +18,21 @@
 # of monit v5.15+ where the issues are fixed.
 
 (
+  exec  >/proc/1/fd/1
```

Comment:
We should log to `/var/vcap/sys/log`.

Reply:
Then we probably want to make sure that directory exists (since there's no actual ordering defined on what order the tasks start in), and possibly ensure that the directory is owned by `vcap` (because IIRC we run as root here).

Huh, wait, `/var/vcap/sys/log` is actually owned by root, with boring permissions (`0755`) on a running container. Okay.

@viovanov Do we want append or overwrite? I could see us wanting both options… append because we want to know if one task succeeded and one failed, and overwrite to avoid very large logs…